### PR TITLE
Added a few abbreviations and sorted the list.

### PR DESCRIPTION
**Pull request by Wandmalfarbe**

### CODE REVIEW

File: data/abbreviations [hunk 1]
```diff
@@ -1,28 +1,54 @@
...
+Mr.
+Mrs.
+Ms.
+No.
+Nov.
+Oct.
+Okt.
 p.
+Ph.D.
+PhD.
```
Comment (tarleb) [inline data/abbreviations]: I've only seen *Ph.D.* or *PhD*, but not *PhD.*. Is this commonly used?

Comment (Wandmalfarbe) [reply data/abbreviations]: I don't think it's commonly used so I removed it. I also removed the german abbreviations except `Sept.`.

**Comment by Wandmalfarbe** [comment]
I left `Sept.` in because it is the English abbreviation for September: <https://english.stackexchange.com/questions/408701/abbreviation-for-september>
<http://public.oed.com/how-to-use-the-oed/abbreviations/#s>

I will add `Jun.`, `Jul.`, and `Sep.` for consistency with the other three letter abbreviated months.

**Comment by tarleb** [comment]
Interesting, didn't know. Thanks!

**Pull request merged**